turbinia/processors/partitions.py

```diff
@@ -40,6 +41,12 @@ def Enumerate(evidence):
   options = volume_scanner.VolumeScannerOptions()
   options.partitions = ['all']
   options.volumes = ['all']
+  if location:
+    log.debug('Parsing for partition at location {0!s}'.format(location))
```

Comment:
Maybe add the evidence name here, something like "Scanning {evidence.name} for partition..."